### PR TITLE
Add Organization CRD

### PR DESCRIPTION
**Pull request by kubasobon**
Towards https://github.com/giantswarm/roadmap/issues/148
Specification: https://intranet.giantswarm.io/docs/architecture-specs-adrs/specs/namespaced-crs/

## Checklist

- [x] Consider SIG UX feedback.
- [x] Update changelog in CHANGELOG.md.
-  ~~If adding a new type, ensure that it is added to the [CRD unit tests](https://github.com/giantswarm/apiextensions/blob/d78aba91d578d56ef49f58e336035d35edc4e1b0/pkg/crd/crd_test.go#L9)~~ _Example is out of date. Is this step necessary?_

### CODE REVIEW

File: pkg/apis/security/v1alpha1/organization_types.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+// +kubebuilder:resource:categories=common;giantswarm
+// +kubebuilder:storageversion
+
+type Organization struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata"`
+	Spec              OrganizationSpec `json:"spec"`
+}
+
+type OrganizationSpec struct {
```
Comment (corest) [inline pkg/apis/security/v1alpha1/organization_types.go:31]: my favorite type of CRs

**Review by corest — APPROVED** [review]
lgtm

**Comment by wtty-fool** [comment]
@giantswarm/sig-ux I'm getting 
```
diff --git a/config/crd/v1beta1/security.giantswarm.io_organizations.yaml b/config/crd/v1beta1/security.giantswarm.io_organizations.yaml
index ea5971f7..8004db0d 100644
--- a/config/crd/v1beta1/security.giantswarm.io_organizations.yaml
+++ b/config/crd/v1beta1/security.giantswarm.io_organizations.yaml
@@ -17,7 +17,7 @@ spec:
     listKind: OrganizationList
     plural: organizations
     singular: organization
-  scope: Cluster
+  scope: Namespaced
```
but I'm using `+genclient:nonNamespaced`. Is there something I should be cognizant of?

**Comment by tfussell** [comment]
> @giantswarm/sig-ux I'm getting
> 
> ```
> diff --git a/config/crd/v1beta1/security.giantswarm.io_organizations.yaml b/config/crd/v1beta1/security.giantswarm.io_organizations.yaml
> index ea5971f7..8004db0d 100644
> --- a/config/crd/v1beta1/security.giantswarm.io_organizations.yaml
> +++ b/config/crd/v1beta1/security.giantswarm.io_organizations.yaml
> @@ -17,7 +17,7 @@ spec:
>      listKind: OrganizationList
>      plural: organizations
>      singular: organization
> -  scope: Cluster
> +  scope: Namespaced
> ```
> 
> but I'm using `+genclient:nonNamespaced`. Is there something I should be cognizant of?

That comment only applies to the generated client (the CRUD interface for using the CR). You'll need another code generation comment for kubebuilder which applies to the CRD: `// +kubebuilder:resource:scope=Cluster,categories=common;giantswarm` See the [Release CR/CRD](https://github.com/giantswarm/apiextensions/blob/master/pkg/apis/release/v1alpha1/release_types.go#L53) for an example.

**Pull request merged**